addons/beehave/nodes/beehave_root.gd

```diff
@@ -4,10 +4,22 @@ class_name BeehaveRoot
 @icon("../../icons/tree.svg")
 
 var Blackboard = load("res://addons/beehave/blackboard.gd")
+
 const SUCCESS = 0
 const FAILURE = 1
 const RUNNING = 2
 
+enum BeehaveProcessMode {
```

Comment:
Is it possible to call this just `ProcessMode`?

Reply:
[ProcessMode ](https://docs.godotengine.org/en/latest/classes/class_node.html?highlight=ProcessMode#enumerations) are reserved enums of "Node". Therefore, I can not use either set_process_mode or ProcessMode

Reply:
Could we then just re-use that one instead of introducing a new one?